file-formats/ddls/ddls-v1.json

```
@@ -129,7 +129,7 @@
         "Projection View"
       ],
       "enumDescriptions": [
-        "View",
+        "DDIC-Based Views",
         "View Entity",
         "View Extend",
         "View Entity Extend",
```

Not really  related to the actual PR. But I see descriptions are title style. They shall be sentence style. This is true for the whole DDLS file format.

```suggestion
        "DDIC-based views",
        "View entity",
        "View extend",
        "View entity extend",
```

@BeckerWdf Do you want to update all descriptions in this PR or shall I create an separate issue?

---

@schneidermic0: Let's handle this in a separate issue.

---

I have just created a new issue: #441